src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1082,6 +1085,40 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
                                                                        bundle.getResponseAnswerHtml(response, question),
                                                                        moderationButton);
             configureResponseRow(prevGiver, response.recipient, responseRow);
+            String giverName = bundle.getNameForEmail(response.giver);
+            String recipientName = bundle.getNameForEmail(response.recipient);
+
+            String giverTeam = bundle.getTeamNameForEmail(response.giver);
+            String recipientTeam = bundle.getTeamNameForEmail(response.recipient);
+
+            giverName = bundle.appendTeamNameToName(giverName, giverTeam);
+            recipientName = bundle.appendTeamNameToName(recipientName, recipientTeam);
+
+            List<FeedbackResponseCommentRow> comments =
+                    buildResponseComments(giverName, recipientName, question, response);
+            if (!comments.isEmpty()) {
+                responseRow.setCommentsOnResponses(comments);
+            }
+            Map<FeedbackParticipantType, Boolean> responseVisibilityMap = getResponseVisibilityMap(question);
+            boolean isCommentsOnResponsesAllowed = question.getQuestionDetails()
+                    .isCommentsOnResponsesAllowed();
```

Comment:
A higher-level break is preferred; try shifting `question.getQuestionDetails()` down to this line as well.